sendgrid.com.domain-auth.json

```diff
@@ -0,0 +1,30 @@
+{
+  "providerId":"sendgrid.com",
+  "providerName":"Twilio SendGrid",
+  "serviceId":"domain-auth",
+  "serviceName":"Twilio SendGrid Domain Authentication",
+  "version": 1,
+  "logoUrl":"https://sendgrid.com/brand/sg-twilio/SG_Twilio_Lockup_RGBx1.png",
+  "syncPubKeyDomain": "domainconnect.sendgrid.net",
+  "syncRedirectDomain": "sendgrid.com",
+  "records":[
```

Comment:
Please add ` "hostRequired": true,`, as now the template defines CNAME at `@` so it must be always applied with host parameter and cannot be at APEX.

Reply:
ok fixed its 🟢  now